en/controllers/middleware.rst

```diff
@@ -386,17 +386,50 @@ backwards compatible with ``CookieComponent`` from earlier versions of CakePHP.
 Cross Site Request Forgery (CSRF) Middleware
 ============================================
 
-CSRF protection can be applied to your entire application, or to specific scopes
-by applying the ``CsrfProtectionMiddleware`` to your middleware stack::
+CSRF protection can be applied to your entire application, or to specific routing scopes.
 
+.. note::
+
+    You cannot use both following approaches together, you must choose only one.
+    If you use both approaches together, CSRF token mismatch error will occur on every `PUT` and `POST` request
+
+
+.. warning::
+
+    You cannot use ``CsrfComponent`` together with ``CsrfProtectionMiddleware``, the warning about conflicting components is not shown until 3.7.0.
```

Comment:
Having a warning and note beside each other seems redundant. Could they be combined?

Reply:
Only if we were to split the documentation to two sections with appropriate headers, then the note would be less important, warning is still necessary